dockerfiles/init/manifests/che.env

```diff
@@ -190,10 +190,12 @@
 # Che Server API Endpoint
 #     The location of the API end point where dashboard and IDE clients will look for
 #     interacting with the Che server, which we also call the workspace master.
-#CHE_API=http://${CHE_HOST}:${CHE_PORT}/api
+#CHE_API=${CHE_HOST_PROTOCOL}://${CHE_HOST}:${CHE_PORT}/api
 # Che Server default websocket endpoint.
 #     Provides basic communication endpoint where clients can get/push statuses/output.
 #CHE_WEBSOCKET_ENDPOINT=ws://${CHE_HOST}:${CHE_PORT}/api/websocket
+# or for HTTPS
+#CHE_WEBSOCKET_ENDPOINT=wss://${CHE_HOST}:${CHE_PORT}/api/websocket
```

Comment:
maybe we should have CHE_HOST_WS_PROTOCOL? 
we can automatically set CHE_HOST_WS_PROTOCOL="wss" if CHE_HOST_PROTOCOL="https"

Reply:
Right now, I will detect `CHE_HOST_PROTOCOL=https` and enable `wss` accordingly. Is there any use case for having `https` with a different setting then `wss`? It should be easy to add `CHE_HOST_WS_PROTOCOL` though if we want to decouple the protocols.